index.rst

```diff
@@ -44,14 +44,579 @@
+Some remain under active development whereas others are considered nearly complete.
+
+LOVE
+''''
+
+.. tiago writing this
+LOVE is a web-based user inteface for the observatory control system. 
+It is currently available at the `summit <http://amor01.cp.lsst.org/>` (VPN required) and all other test stands.
+
+The system is developed using the popular React JavaScript framework for the frontend, with Python in the backend.
+It contains an editing tool that allow users to create and customize views, but this feature rely exclusively on existing widget-like components.
+Extending the system beyond the vendor-provided "widgets" would require some knowledge of React and JavaScript, which is not common in the project.
+Nevertheless, LOVE does contain a especial widget that allows us to embed other web pages into a particular view. 
+This could be used, for instance, to embed `Bokeh Plotting Applications`_ alongside other LOVE components.
+
+
+
+Nublado (Jupyter Notebook) Interface
+''''''''''''''''''''''''''''''''''''
+
+.. patrick writing this
+The summit `Nublado interface <https://summit-lsp.lsst.codes/>`_ (VPN required) provides the user with a Jupyter Lab interface and the required libraries/packages to perform standard observatory operations including sending commands and running SAL scripts.
+It can also be used to query the EFD.
+This tool is setup to mimick the `RSP`_ and test-stand environments to the maximum extent possible, providing all the functionalities of a Jupyter Notebook but with direct access to the control system.
+Observers are expected to use this tool to perform commands or sequences that are not encapsulated into a SAL script.
+They are also useful for on-the-fly analysis and/or custom monitoring.
+
+
+Camera Visualization Tool
+''''''''''''''''''''''''''
+
+.. tony writing this
+
+Engineering Facilities Database and Large File Annex
+'''''''''''''''''''''''''''''''''''''''''''''''''''''
+.. Patrick writing this
+
+The `Engineering Facilities Database (EFD) <https://sqr-034.lsst.io/#introduction>`_ records all commands, events, and telemetry sent over the DDS control network.
+This content essentially tracks the observatory state as a function of time and is very useful in diagnosing issues and understanding (both desired and undesired) operational behaviours.
+The database is best queried using the `EFD client <https://efd-client.lsst.io/>`_ from the `Nublado (Jupyter Notebook) Interface`_ (or any python-based method/script) when making custom plots.
+Accessing the EFD, and specifically the other instances of the data, is found in `SQR-034 <https://sqr-034.lsst.io/#efd-deployments>`_.
+However, the `Chronograf`_ graphical front-end offers a nice solution for building simple plots (dashboards).
+
+The `Large File Annex (LFA) <https://tstn-029.lsst.io/>`_ contains files over ~1 MB that are accessible both from the summit and the `RSP`_.
```

Comment:
Is this referring to a particular instance of the RSP on the summit or another location?

Reply:
Ah. Thank you for clarifying that distinction. That would have been useful information for me reading this report, but maybe for others it is already clear.